lib/applicants/verify-identity/validation.ts

```diff
@@ -0,0 +1,17 @@
+import { object, string, date, number } from 'yup';
+
+/**
+ * Verify identity validation schema
+ */
+export const verifyIdentitySchema = object({
+  userId: number()
+    .typeError('User ID must be a positive number')
+    .required('Please enter your User ID')
+    .positive('Please enter your User ID')
```

Comment:
@angeladietz
The initial value of `userId` is 0 so the message `User ID must be a positive number` would pop up (since zero is considered a non-positive number) when the user doesn't fill in the field so I felt the `Please enter your User ID` might be clearer in this scenario. Lmk if you think there's a better way to address this!

<img width="399" alt="image" src="https://user-images.githubusercontent.com/51224641/159342663-6dd3706d-42d8-4812-a060-d8b3fe801857.png">

Reply:
This sounds good to me